### PR TITLE
Fix bug and add example

### PR DESCRIPTION
**Pull request by smajee**
- Fixes bug in scico.svmbir that caused errors with the new release of svmbir
- Fixes issue #198 by adding an example that uses the svmbir extended losss

### CODE REVIEW

File: examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py [hunk 1]
```diff
@@ -85,7 +96,7 @@
 
 
 """
-Set up an ADMM solver.
+Set up an ADMM solver with explicit NonNegativeIndicator
```
Comment (bwohlberg) [inline examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py:99]: Nitpick: missing terminating period (not only here).

Comment (smajee) [reply examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py]: Added the missing periods

**Review by bwohlberg — APPROVED** [review]
Looks good, but please rebuild the notebook corresponding to the modified script and update the submodule in this branch.

**Comment by smajee** [comment]
> Looks good, but please rebuild the notebook corresponding to the modified script and update the submodule in this branch.

Done.

**Pull request merged**